app/services/return-requirements/check/returns-requirements.service.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/**
+ * Orchestrates fetching and presenting the return requirements for the check page
+ * @module ReturnRequirementsService
+ */
+
+const ReturnRequirementsPresenter = require('../../../presenters/return-requirements/check/returns-requirements.presenter.js')
+const SessionModel = require('../../../models/session.model.js')
+const PurposeModel = require('../../../models/purpose.model.js')
+
+/**
+ * Orchestrates fetching and presenting the return requirements for `/return-requirements/{sessionId}/check` page
+ *
+ * @param {string} sessionId - The UUID for return requirement setup session record
+ *
+ * @returns {Promise<Object>} page data needed by the view template
+ */
+async function go (sessionId) {
+  const session = await SessionModel.query().findById(sessionId)
```

Comment:
As this is called by `CheckService` which has already fetched the `SessionModel` instance we can pass that in instead of the ID and avoid making the query again.

I appreciate every other service in return requirements fetch the session. But this is not because they are following a convention for services. Simply the other pages are simple enough a single service is all that is needed.

Reply:
Have addressed and updated the tests